slam-libraries/viam-orb-slam3/orbslam_server_v1.h

```diff
@@ -39,6 +41,10 @@ class SLAMServiceImpl final : public SLAMService::Service {
     ::grpc::Status GetMap(ServerContext *context, const GetMapRequest *request,
                           GetMapResponse *response) override;
 
+    ::grpc::Status GetPointCloudMap(
```

Comment:
Can we add comments to at the very least the GRPC functions?

Reply:
done